### DISCUSSION TITLE
Formalize the deck.gl support

### DISCUSSION THREAD

**Discussion by hy9be** [maplibre/maplibre-gl-js#2919] [Ideas]
deck.gl has a big data visualization community. But its developer guide only has a short and out-of-date description about MapLibre as a Mapbox v1 fork https://deck.gl/docs/developer-guide/base-maps/using-with-mapbox#compatibility-with-mapbox-gl-js-forks. It does not help the users who want to use deck.gl with maplibre-gl-js.

Suggest we officially connect with the deck.gl community to 

1. Create a designated page for MapLibre under the ``Using with Supported Base Maps`` section of their developer guide like https://deck.gl/docs/developer-guide/base-maps/using-with-mapbox. It should include up-to-date and detailed usage instructions and known issues.
2. Aligned on the long-term roadmap so that the two libraries are always compatible.

**Comment by HarelM** [comment]
Cc: @Pessimistress
You can add an example in this repo that shows how to use deck.gl with maplibre, you can also open a PR in the deck.gl repo to update the docs or add examples with maplibre.
Generally speaking, @Pessimistress had joined some TSC meetings and contributed some code, our representatives joined a deck.gl conference last year and we are generally in alignment and with good faith.

**Comment by hy9be** [comment]
Talked with @Pessimistress offline. We have some action items now.
